components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -408,7 +408,7 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
             }
             if (code !== 200) {
                 // getting workspace auth didn't work as planned
-                console.error("Unable to retrieve workspace-auth cookie! Quitting.", {
```

Comment:
should we check here that it is not retriable and 404?

or you think generally is alright?

Reply:
It's not well documented, but this branch (`retry: false`) is used in places where we optimistically try to fetch the cookie early without requiring it. Only later we call the method with `retry: true` and `await` it.

We could re-validate the whole function and fetch mechanism because since it's inception, a lot of things changed:
 - the returny-early case does not work anymore since the cookie is http-only now ([link](https://github.com/gitpod-io/gitpod/pull/18772/files#diff-d897e36a784992a6a4d3b76c09147a96375add6c80bd14dca49657dcff06b3a7R371))
 - timings on workspace events changed so much that I wonder whether we should call it in different places

Still, it works, and I'm only focussed on cosmetics right now. Fine with you?